lib/dk_buttons_and_leds/dk_buttons_and_leds.c

```diff
@@ -102,9 +101,7 @@ int dk_buttons_init(button_handler_t button_handler)
 {
 	int err;
 
-	if (button_handler != NULL) {
-		button_handler_cb = button_handler;
-	}
+	button_handler_cb = button_handler;
```

Comment:
Couldn't this be `NULL`?

Reply:
If this is null the callback would not be called. I do not see any reason for this conditional write like above.